### ISSUE TITLE
App Settings iFrame

### ISSUE THREAD

**Issue by JannikZed** [mirumee/saleor-dashboard#934]
### What I'm trying to achieve
I can register Third-Party apps. Now I wanted to create a settings page that gets rendered as iFrame in the Dashboard.But I can't get it running, as it is not a normal iFrame and there are no docs concerning that feature.
What exactly do you expect to come from our server? I can see something like a script parser, but I really don't know what exactly I need to build in order to show our App settings in the dashboard.

### Steps to reproduce the problem
1. 
2. 

### What I expected to happen
…

### Http request and response from inspector

```
// insert request and response here 
```

### Console output

```
// insert console output here 
```

**System information**

Operating system: 
- [ ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Other 

Browser:
- [ ] Safari
- [ ] Chrome
- [ ] Edge
- [ ] Other

**Comment by JannikZed** [comment]
so for people trying to use Next.Js: a possibility is to set the "asset-prefix-url" to the URL of your service and furthermore add CORS header to the next.config.js.
Nevertheless, I still can't use Next.js production build, as the iframe implementation is really buggy in the dashboard. The "AppDetailsSettingsPage" is failing here: `const formURL = new URL(documentElement.querySelector("script").src);`

**Comment by krzysztofwolski** [comment]
Bug resolved in 3.0
